draft-ietf-quic-load-balancers.md

```diff
@@ -926,6 +926,27 @@ connection IDs to the same server.  The QUIC-LB algorithms do prevent the
 linkage of two connection IDs to the same individual connection if servers make
 reasonable selections when generating new IDs for that connection.
 
+## Limited configuration deployment
+
+A naive deployment of QUIC-LB in a cloud provider might use the same global
+QUIC-LB configuration across all the load balancers in its enterprise. An
```

Comment:
I don't think that the scope of this is very clear.  If the idea is that the cloud provider - the one who provides the load balancer - provides the same configuration to mutually distrustful customers, then there is a very important point to make.  But I'm not sure that this is clear from the text here.  "enterprise" isn't well connected to "cloud provider".

Reply:
OK, I got rid of the word 'enterprise' and I think it's clearer now.